.jsonschema.json

```diff
@@ -29,6 +24,35 @@
           "description": "The brand guidelines",
           "$ref": "#/definitions/url"
         },
+        "aliases": {
```

Comment:
As it could, potentially, be a very large object and the information it contains is less important than the licensing info, I'd move the `aliases` entry to the end, after the `license` entry.

Reply:
Can you elaborate? Do you mean the aliases should come after the license in the data file? If so, I agree but the ordering here does not enforce an order in the data file. I put it before the licenses here because this schema shorter than the licences schema.

Reply:
Ah, OK, I always thought this was where the ordering was enforced. Ignore that, so! 😆